how-to/install/multimc.md

```diff
@@ -0,0 +1,67 @@
+   * [`1.21.1`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v6.3.0/Fabulously.Optimized.MC.1.21.1.auto-update.zip)
+   * [`1.20.6`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.12.0-beta.11/Fabulously.Optimized.MC.1.20.6.auto-update.zip)
+   * [`1.19.4`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v4.10.0/Fabulously.Optimized.MC.1.19.4.auto-update.zip)
+   * [`1.18.2`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.12.2/Fabulously.Optimized.MC.1.18.2.auto-update.zip)
+   * [`1.17.1`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.17.1.auto-update.zip)
+   * [`1.16.5`](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.16.5.auto-update.zip)
+3. If you do not want the latest version, you can find older ones in [GitHub Releases](https://github.com/Fabulously-Optimized/fabulously-optimized/releases)
+4. Open [**MultiMC**](https://multimc.org/)
+5. Drag the `zip` file and drop it inside of MultiMC's window. A window should pop up
+6. Click **OK** in that window. Something should happen <!-- TODO: what though?? -->
+7. Double-click on that window. The download should start
+8. If you are asked to download a mod manually:
+   1. Open the given address in your browser. The mod's page should open up
+   2. Click on **Cancel Launch**
+   3. Click on **Download**. A `jar` file should be downloaded
+   4. Go back to **MultiMC**
+   5. Click on the instance you just created
+   6. Click on **View Mods**
+   7. Drag the mod you downloaded in _step 8.3_ and drop it in the mod list
+9. Click on **Launch**. Minecraft should open up
+10. If you can see `Fabulously Optimized` in the bottom-right corner, you're done!
+
+## How Does It Work?
+
+The MultiMC auto-updating pack launches [`packwiz`](https://github.com/comp500/packwiz) every time it is opened. `packwiz` is a mod handling tool which will add and update any missing FO mods indicated in a `pack.toml` file.
+
+Since the update checker runs on every launch, you'll get a skippable error if you play offline. The updater only manages FO's mods, not the [mods added manually](../add-mods/multimc.md). The updater does not work across Minecraft versions, so you'll need to install it again every time.
+
+MultiMC does not natively support automatic updates of packs: see [issue #2640](https://github.com/MultiMC/MultiMC5/issues/2640) and [issue #3037](https://github.com/MultiMC/MultiMC5/issues/3057). FO chose to use `packwiz` because it is also used for the [vanilla installer](vanilla.md).
+
+Before FO 6.0.0, the MultiMC auto-updating pack was distributed on CurseForge, but that has changed for the following reasons:
```

Comment:
This variant was never on CurseForge, the original MultiMC pack was. Though I don't think it is even necessary to mention it here.